meson.build

```
@@ -6,8 +6,13 @@ lib = static_library('blurhash',
            'blurhash.cpp',
            install : true)
 
-doctest_proj = subproject('doctest')
-doctest_dep = doctest_proj.get_variable('doctest_dep')
+doctest_dep = dependency('doctest', required : false)
+
+if not doctest_dep.found()
+	doctest_proj = subproject('doctest')
+	doctest_dep = doctest_proj.get_variable('doctest_dep')
+endif
+
```

```suggestion
doctest_dep = dependency('doctest', fallback : ['doctest', 'doctest_dep'])

```

Should do the same right? In theory we could also use a wrap with a provides section like here: https://mesonbuild.com/Wrap-dependency-system-manual.html#provide-section But that probably is too much to change atm, so the above is fine for now.

---

Did not know about fallback, tested both cases and it build just fine, applied and squashed